promgen/management/commands/bootstrap.py

```diff
@@ -58,3 +59,14 @@ def handle(self, **kwargs):
         # Schemes based on list of supported brokers
         # http://docs.celeryproject.org/en/latest/getting-started/brokers/index.html
         self.write_setting('CELERY_BROKER_URL', test=URLValidator(schemes=['redis', 'amqp', 'sqs']))
+
+        if User.objects.filter(is_superuser=True).count() == 0:
```

Comment:
I just realized that bootstrap is often run before migrate so this would cause problems. I may need to have bootstrap automatically call migrate which would help test the settings and ensure that we're able to create the super user.